pulsar-common/src/main/java/org/apache/pulsar/common/protocol/Commands.java

```diff
@@ -440,6 +440,17 @@ public static void skipMessageMetadata(ByteBuf buffer) {
         buffer.skipBytes(metadataSize);
     }
 
+    public static long getEntryPublishTimestamp(ByteBuf headersAndPayloadWithBrokerEntryMetadata) throws IOException {
```

Comment:
change the method name to `getEntryTimestamp` is better.

Reply:
fix